api/web3server.go

```diff
@@ -41,6 +41,7 @@ type (
 	}
 
 	web3Handler struct {
+		cfg         Config
```

Comment:
can just add `batchSizeLimit`, since web3handler does not need the entire `Config`?

Reply:
done